modules/API/API.js

```diff
@@ -43,198 +36,239 @@ let postis;
+        'video-hangup': () => APP.conference.hangup(),
+        'email': APP.conference.changeLocalEmail,
+        'avatar-url': APP.conference.changeLocalAvatarUrl,
+        'remote-control-event': event =>
             APP.remoteControl.onRemoteControlAPIEvent(event)
     };
-    Object.keys(commands).forEach(function (key) {
+    Object.keys(commands).forEach(key => {
         postis.listen(key, args => commands[key](...args));
     });
 }
 
 /**
  * Sends message to the external application.
- * @param message {object}
- * @param method {string}
- * @param params {object} the object that will be sent as JSON string
+ *
+ * @param {Object} message - The message to be sent.
+ * @returns {void}
  */
 function sendMessage(message) {
-    if(enabled) {
+    if (enabled) {
         postis.send(message);
     }
 }
 
 /**
  * Check whether the API should be enabled or not.
+ *
  * @returns {boolean}
  */
-function shouldBeEnabled () {
-    return (typeof jitsi_meet_external_api_id === "number");
+function shouldBeEnabled() {
+    return typeof jitsiMeetExternalApiId === 'number';
 }
 
 /**
  * Sends event object to the external application that has been subscribed
  * for that event.
- * @param name the name event
- * @param object data associated with the event
+ *
+ * @param {string} name - The name event.
+ * @param {Object} object - Data associated with the event.
+ * @returns {void}
  */
-function triggerEvent (name, object) {
-    if(enabled) {
-        sendMessage({method: name, params: object});
+function triggerEvent(name, object) {
+    if (enabled) {
+        sendMessage({ method: name,
+            params: object });
     }
 }
 
+/**
+ * Implements API class that communicates with external api class
+ * and provides interface to access Jitsi Meet features by external
+ * applications that embed Jitsi Meet
+ */
 class API {
     /**
-     * Constructs new instance
-     * @constructor
-     */
-    constructor() { }
-
-    /**
      * Initializes the APIConnector. Setups message event listeners that will
      * receive information from external applications that embed Jitsi Meet.
      * It also sends a message to the external application that APIConnector
      * is initialized.
-     * @param options {object}
-     * @param forceEnable {boolean} if true the module will be enabled.
+     *
+     * @param {Object} options - Optional parameters.
+     * @param {boolean} options.forceEnable - If true the module will be
+     * enabled.
+     * @returns {void}
      */
-    init (options = {}) {
-        if(!shouldBeEnabled() && !options.forceEnable)
+    init(options = {}) {
+        if (!shouldBeEnabled() && !options.forceEnable) {
             return;
+        }
 
         enabled = true;
 
-        if(!postis) {
+        if (!postis) {
             this._initPostis();
         }
     }
 
     /**
-     * initializes postis library.
+     * Initializes postis library.
+     *
+     * @returns {void}
+     *
      * @private
```

Comment:
I think it's desired for the annotations to be in alphabetical order, so private goes prefer returns.